runtime/src/governance/proposals.rs

```diff
@@ -143,6 +143,9 @@ decl_event!(
 
         /// * Hash - hash of wasm code of runtime update.
         RuntimeUpdated(u32, Hash),
+
+        /// Root cancelled proposal
+        ProposalVeto(u32),
```

Comment:
It makes sense to add the second param to this event: AccountId - who vetoed.

Reply:
since it is the root origin not sure it adds additional useful information.